### DISCUSSION TITLE
Displaying the contents of .nvmrc

### DISCUSSION THREAD

**Discussion by Daeraxa** [JanDeDobbeleer/oh-my-posh#3575] [Troubleshoot] [answered]
I've edited my prompt to include the [node](https://ohmyposh.dev/docs/segments/node) segment which works as I expect except what I kind of want to see isn't necessarily the version of node that I'm currently using but the version of node specified in the file.

For example if I have an .nvmrc which has `16` then I want my prompt to display this if I'm entering the folder as well as the active version. I know that I can use `.mismatch` to display if there is a difference in the content of what was read from `.nvmrc` and my active node version but it doesn't display what it actually is.

```json
        {
          "type": "node",
          "foreground": "green",
          "properties": {
            "fetch_package_manager": true
          },
          "template": "{{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Major }}"
        },
```

This is the block I currently have which works but it displays the active node version and not the `.npmrc` version.

For example what I might want in the template (if it existed) would be:

```
{{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Major }} \uE718 {{ .nvmrc }}
```

Any ideas?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Daeraxa that property isn't exposed but can be, it's only a feature request away!

**Reply by Daeraxa** [reply]
Thanks, I'll add one in (and maybe have a look at the same time).
